client/src/controllers/MessagesController.ts

```diff
@@ -0,0 +1,46 @@
+import { Controller } from '@hotwired/stimulus';
+
+export class MessagesController extends Controller<HTMLFormElement> {
+  static classes = ['newMessage', 'show'];
+  static targets = ['messages', 'templates'];
+
+  declare readonly hasMessagesTarget: boolean;
+  declare readonly hasNewMessageClass: boolean;
+  declare readonly hasShowClass: boolean;
+  declare readonly hasTemplatesTarget: boolean;
+  declare readonly messagesTarget: HTMLElement;
+  declare readonly newMessageClass: string;
+  declare readonly showClass: string;
+  declare readonly templatesTarget: HTMLElement;
+
+  add(event) {
+    const status = event.detail.status;
+    const statusText = event.detail.text;
+    // console.log("connected add.",status,statusText);
+    // console.log(this.templatesTarget);
+    if (this.hasNewMessageClass) {
```

Comment:
Check what happens here if multiple messages are added, make sure it doesn't keep adding the class. 

Additionally, we may want to add a value to track the messages added count. This intentionally will change the DOM and provide a nice hook for other code or styles. 

E.g. this.messagesAddedValue ++;